### PR TITLE
fix: dark mode, mobile, homepage design and minor cleanup 

### PR DESCRIPTION
**Pull request by ndom91**
- Add darkmode support
- Add mobile support
- Cleanup marquee a bit and add gradient to side
- Slightly redesign homepage tabs/code component
- Homepage primary action btn 
- Add homepage footer
- Nicer scroll bars (on linux/windows at least haha)
- Add slightly moving homepage blur animation to bg
- Cleanup spelling and simplify many components / pages

<table>
<tr>
<td>Dark</td>
<td>Light</td>
<td>Mobile</td>
<td>Mobile</td>
</tr>
<tr>
<td>

image

</td>
<td>

image

</td>
<td>

image

</td>
<td>

image

</td>
</tr>
</table>

> Not super happy with the blur bg thing in dark mode yet, although it does look better in real life than in the screenshot imo. But still not great.. oh well just an idea, we can always tweak it or do something else.

Follow up ideas for homepage:
- Maybe another section between marquee / footer to include some more high-level info and links off to example projects like Luis had previously. Something like what https://docs.dyte.io/ has maybe? :shrug:

### CODE REVIEW

**Comment by ndom91** [comment]
Little bonus, I got the GH Star Counter thign working again :blush:

image

It's positioned absolute relative to an element that'll always be right next to the nextra built-in GH logo-link, so we should be good. Also it'll default to `20k`, in case the network request doesn't work or somethign, it'll be ~close anyway. The network request is happening in the `Footer` component, which is on every page so that should be fine too :crossed_fingers:

**Comment by ndom91** [comment]
Also added initial Clerk text/link back to homepage

Auth.js homepage mockup


---

What do yall think of these alternatives though? :thinking: They all have the Clerk link "above the fold", so I think they'd probably be alright with that, no?

<table>
<tr>
<td>

image


</td>
<td>

image


</td>
<td>

image


</td>
</tr>
</table>

File: pages/overview/authentication.mdx [hunk 1]
```diff
@@ -3,17 +3,24 @@ import { Link } from "@/components/Link";
...
 
 # Authentication
 
-Before start setting up Auth.js, you need to decide how you're gonna authenticate users in your application. Auth.js supports <u>several authentication paradigms</u>:
+<Callout>
+  Auth.js discourages the use of passwords due to the inherent security risks of
+  the username-password model. Passwords are still supported through the
+  [external auth]((#setup-with-external-auth)) but we strongly encourage to
+  setup a combination of **OAuth + Passwordless** for authentication.
+</Callout>
```
Comment (0ubbe) [inline pages/overview/authentication.mdx]: This one we actually need to remove, @balazsorban44 suggested is a bit of a polemical warning, fine to keep it in the **Credentials** tab 👍🏽

File: pages/overview/authentication.mdx [hunk 2]
```diff
@@ -3,17 +3,24 @@ import { Link } from "@/components/Link";
...
+
+Before start setting up Auth.js, you need to decide how you're gonna authenticate users in your application. Auth.js supports <u>three main authentication paradigms</u>:
 
 - **OAuth** (_Sign in with Google, Github, LinkedIn, etc..._)
-- **Token-based** ( _Magic links, OTP codes, etc..._ )
-- **Credentials** ( _External Auth, Webauthn, etc..._ )
+- **Passwordless** (_Magic links, WebAuthn, etc..._)
```
Comment (0ubbe) [inline pages/overview/authentication.mdx]: Should be Token-based, maybe some issue when rebasing `main` ?

Comment (ndom91) [reply pages/overview/authentication.mdx]: Hmm yeah odd. Fixed this and removed the call out you mentioned above :+1:

**Review by ThangHuuVu — APPROVED** [review]

**Pull request merged**